source/auditing-logging.txt

```diff
@@ -192,3 +252,51 @@ In addition to the following examples, see the blogpost
       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
       .. include:: /includes/examples/tf-example-auditing-filter.rst
+
+   .. tab:: Admin API
```

Comment:
Can we remove the Admin API tab and instead after the TF example explain you can't use TF to return logs, and then add links to the API docs for those? Not full examples with the endpoint but just links, in the TF tab.